vpdq/cpp/hashing/filehasher.h

```diff
@@ -42,6 +42,35 @@ bool hashVideoFile(
     const int height,
     const double framesPerSec,
     const char* programName);
+
+/**
+ * Get frames by passing video file through ffmpeg
+ * Then get pdq hashes for selected frames every secondPerHash
+ *
+ * @param inputVideoFileName Input video's name
+ * @param pdqHashes Vector which stores hashes
+ * @param ffmpegPath Ffmpeg's path
+ * @param verbose If produce detailed output for diagnostic purposes
+ * @param secondsPerHash The time period of picking frames in vpdq
+ * @param width Specified width of the input video
+ * @param height Specified height of the input video
+ * @param framesPerSec Frames per second of the input video
+ * @param downsampleFrameDimension The down-scale dimensions for vpdq
+ * @param programName The name of executable program which invokes the function
+ *
+ * @return If successfully hash the video
+ */
+
+bool hashVideoFileFFMPEG(
```

Comment:
Yes, I think it should be removed, but that requires resolving the issue with the Python binding. It should be removed in a separate issue in my opinion. I was working on getting the Python binding linked with libav*, but I'm not very experienced in build systems. I'm trying to figure out a way to utilize CMake so that I don't have to directly write the path to the libraries.